src/components/Chat/ChatList.tsx

```diff
@@ -89,6 +102,78 @@ export function ChatList() {
       ).length
     : 0;
 
+  const activeIds = !!data
+    ? data
+        .filter(
+          (d) =>
+            d.chatMembers.find((cm) => cm.userId === currentUser?.id)?.status ===
+            ChatMemberStatus.Joined
+        )
+        .map((d) => d.id)
+    : [];
+
+  const activeCount = !!chatCounts
+    ? chatCounts
+        .filter((cc) => activeIds.includes(cc.chatId))
+        .map((cc) => cc.cnt)
+        .reduce((acc, val) => acc + val, 0)
+    : 0;
```

Comment:
I think we could do this in a single iteration instead of three
```suggestion
  const activeCount = !!chatCounts
    ? chatCounts
        .reduce((acc, val) => {
          if (activeIds.includes(cc.chatId)) return acc + val.cnt;
          return acc;
        }, 0)
    : 0;
```